api/src/main/java/org/apache/cloudstack/api/response/CapabilitiesResponse.java

```diff
@@ -140,6 +140,10 @@ public class CapabilitiesResponse extends BaseResponse {
     @Param(description = "true if dynamically scaling for instances is enabled", since = "4.21.0")
     private Boolean dynamicScalingEnabled;
 
+    @SerializedName(ApiConstants.ADDITONAL_CONFIG_ENABLED)
+    @Param(description = "true if additional configurations or extraconfig can be passed to Instances", since = "4.20.2")
```

Comment:
Not related to this PR but as some point we should start using a generic Map response or some other means. Adding a new response param for a new global setting value is an unnecessary hassle